index.js

```diff
@@ -339,16 +352,140 @@ class Raspi extends events.EventEmitter {
     }
   }
 
+  i2cConfig(delay) {
+    if (this[i2c] === undefined) {
```

Comment:
Ok, now I get it.

`execSync('gpio load i2c')` is currently needed for loading module `i2c-dev` which is required for userspace access to I2C. If the `execSync` call is removed, `i2c-dev` will have to be added to /etc/modules manually, or `execSync('gpio load i2c')` will have to be called at the appropriate point in time.